package.json

```diff
@@ -24,7 +24,7 @@
     "lint": "yarn lint:es && yarn lint:style",
     "lint:es": "eslint 'packages/*/src/**/*.js'",
     "lint:style": "stylelint 'packages/*/src/**/*.scss' --report-needless-disables --report-invalid-scope-disables",
-    "test": "jest",
+    "test": "lerna run test --stream && jest",
```

Comment:
@carbon-design-system/ibm-cloud-cognitive-development I believe this will cause duplication for any package that also runs `jest` further downstream, so I'd like to discuss what a good approach for this might be